### PR TITLE
add bets index

### PR DESCRIPTION
**Pull request by tapajos**

### CODE REVIEW

File: app/controllers/bets_controller.rb [hunk 1]
```diff
@@ -2,6 +2,13 @@ class BetsController < ApplicationController
 
   before_action :find_bet
 
+  # GET /users
```
Comment (rodrigotassinari) [inline app/controllers/bets_controller.rb]: i think `/bets` would be better, we are listing and ranking the bets, the user is an "attribute" of the bet

File: app/models/bet.rb [hunk 2]
```diff
@@ -14,6 +14,10 @@ class Bet < ActiveRecord::Base
     presence: true,
     numericality: { only_integer: true, greater_than_or_equal_to: 0, allow_blank: true }
 
+  def score
+    match_bets.sum(:points) + question_bets.sum(:points)
```
Comment (rodrigotassinari) [inline app/models/bet.rb:18]: there's a `points` attribute on the `bets` table, it should just read from it. we have to use the `points` attribute to sort the bets

Comment (tapajos) [inline app/models/bet.rb:18]: hum... where is it being updated? I think it isn't.

File: app/controllers/bets_controller.rb (hunk 1 above)
Comment (tapajos) [inline app/controllers/bets_controller.rb]: Actually it is /bets, the comment is outdated. Fixing it now.

File: app/models/bet.rb (hunk 2 above)
Comment (rodrigotassinari) [inline app/models/bet.rb:18]: it is not YET ;)

File: app/controllers/bets_controller.rb (hunk 1 above)
Comment (tapajos) [inline app/controllers/bets_controller.rb]: fixed

**Comment by rodrigotassinari** [comment]
:+1:

**Pull request merged**